docs/sink-configuration.md

```diff
@@ -88,6 +88,7 @@ The following options are available:
 * `batchSize`- How many metrics are sent in each request to Hawkular-Metrics (default is 1000)
 * `concurrencyLimit`- How many concurrent requests are used to send data to the Hawkular-Metrics (default is 5)
 * `labelTagPrefix` - A prefix to be placed in front of each label when stored as a tag for the metric (default is `labels.`)
+* `disablePreCache` - Disable cache initialization by fetching metric definitions from Hawkular-Metrics
```

Comment:
This does not disable the caching itself, only the preloading from Hawkular-Metrics. For that reason, the `disableCache` would be quite misleading.